src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/csip/CSIPDpsApiServiceTest.kt

```diff
@@ -137,5 +138,64 @@ internal class CSIPDpsApiServiceTest {
         }
       }
     }
+
+    @Nested
+    @DisplayName("PUT /sync/csip-records/move")
```

Comment:
Not sure why you need this test, is it the same as the Integration test? Is there really an endpoint called ""PUT /sync/csip-records/move" ? since this seems to call the service directly